packages/now-cli/src/commands/env/add.ts

```diff
@@ -29,22 +32,25 @@ export default async function add(
   args: string[],
   output: Output
 ) {
+  // improve the way we show inquirer prompts
+  require('../../util/input/patch-inquirer');
```

Comment:
If I remember correctly, this confused users because it removed the comments about `a` for all and `space` for selection in the multi-select.

Is it still necessary?

Reply:
I can add it back and keep the "custom styles".

Just tried it and it looks like this:
![image](https://user-images.githubusercontent.com/6616955/98257221-b4d38200-1f7f-11eb-9a46-28f4475709c2.png)

Reply:
The reason I added it is because we also show this prompt:
![image](https://user-images.githubusercontent.com/6616955/98257303-d0d72380-1f7f-11eb-89f7-b5bb06f9fe76.png)

And I want to make it consistent with the other prompts.

Reply:
I'm a little confused. Which of the 3 screenshots is _with_ the patch and which of the 3 is _without_ the patch?

Reply:
The 3 screenshots up there are _with_ the patch.

I pushed a commit to improve the "checkbox" type prompt here: https://github.com/vercel/vercel/pull/5372/commits/d396c15330f4c02c82b327c9c3446b5dac6e8149

Currently, this is how selecting the environment looks:
![](https://user-images.githubusercontent.com/6616955/98257221-b4d38200-1f7f-11eb-9a46-28f4475709c2.png)

@styfle Does that lgty?